docs/swagger/swagger.yaml

```diff
@@ -4119,6 +4119,79 @@ paths:
           description: Success
         default:
           description: ""
+  /namespaces/{ns}/tokens/{connector}/pools:
```

Comment:
We had a conversation one time that ended with "don't use 'token type' anywhere because it may be a loaded term". I agree `connector` is probably not correct here though.

Reply:
Yeah, think in this particular case we want the human doing the config, to set this (via the connector `name` in the config) to be an actual type... so just here (not in the DB or config) feels like `type` is good.